third_party/2and3/decorator.pyi

```diff
@@ -0,0 +1,90 @@
+# Stubs for decorator (Python 2 and 3)
+#
+# NOTE: This dynamically typed stub was automatically generated by stubgen.
+
+import sys
+from typing import Any, Callable, Dict, List, NamedTuple, Optional, Pattern, Text, Tuple, TypeVar
+
+_C = TypeVar("_C", bound=Callable[..., Any])
+_Func = TypeVar("_Func", bound=Callable[..., Any])
+_Caller = TypeVar("_Caller", bound=Callable[..., Any])
+
+def get_init(cls): ...
+
+# Duplicated from `stdlib/3/inspect.pyi`
+FullArgSpec = NamedTuple('FullArgSpec', [('args', List[str]),
+                                         ('varargs', Optional[str]),
+                                         ('varkw', Optional[str]),
+                                         ('defaults', tuple),
+                                         ('kwonlyargs', List[str]),
+                                         ('kwonlydefaults', Dict[str, Any]),
+                                         ('annotations', Dict[str, Any]),
+                                         ])
+
+if sys.version_info >= (3,):
+    iscoroutinefunction: Callable[[Callable[..., Any]], bool] = ...  # really inspect.iscoroutinefunction
+    getfullargspec: Callable[[object], FullArgSpec] = ...  # really inspect.getfullargspec
+else:
+    def iscoroutinefunction(f: Callable[..., Any]) -> bool: ...
+    def getfullargspec(func: object) -> FullArgSpec: ...
```

Comment:
`getfullargspec()` only accepts functions as arguments. I don't think there is currently a way to express this in typing (`types.FunctionType` would probably be correct, but does not work), so use `Any` to mark this. (It's incorrect in `inspect.pyi` as well.)

```suggestion
    def getfullargspec(func: Any) -> FullArgSpec: ...
```